pitest-aggregator/src/main/java/org/pitest/aggregate/ReportAggregator.java

```diff
@@ -132,6 +137,18 @@ public static class Builder {
     private final Set<File>      mutationResultsFiles    = new HashSet<>();
     private final Set<File>      sourceCodeDirectories   = new HashSet<>();
     private final Set<File>      compiledCodeDirectories = new HashSet<>();
+    private Charset inputCharset;
+    private Charset outputCharset;
+
+    public Builder inputCharSet(Charset inputCharset) {
```

Comment:
@hcoles Working on the Gradle integration, I've noticed that you mix "charset" and "charSet" for fields, "setters" and getters. Maybe they all could switched to just "charset" (with deprecating the old methods)?